docs/docs/core/00-baseapp.md

```diff
@@ -184,30 +185,28 @@ Again we want to highlight that the described behavior is that of the default ha
 
 ![ProcessProposal](./baseapp_state-processproposal.png)
 
-### BeginBlock State Updates
+### FinalizeBlock State Updates
 
-During `BeginBlock`, the `deliverState` is set for use in subsequent `DeliverTx` ABCI messages. The
-`deliverState` is based off of the last committed state from the root store and is branched.
-Note, the `deliverState` is set to `nil` on [`Commit`](#commit).
+During `FinalizeBlock`, the `finalizeBlockState` is set for use during transaction execution and endblock. The
+`finalizeBlockState` is based off of the last committed state from the root store and is branched.
+Note, the `finalizeBlockState` is set to `nil` on [`Commit`](#commit).
 
-![BeginBlock](./baseapp_state-begin_block.png)
-
-### DeliverTx State Updates
-
-The state flow for `DeliverTx` is nearly identical to `CheckTx` except state transitions occur on
-the `deliverState` and messages in a transaction are executed. Similarly to `CheckTx`, state transitions
-occur on a doubly branched state -- `deliverState`. Successful message execution results in
-writes being committed to `deliverState`. Note, if message execution fails, state transitions from
+The state flow for transcation execution is nearly identical to `CheckTx` except state transitions occur on
+the `finalizeBlockState` and messages in a transaction are executed. Similarly to `CheckTx`, state transitions
+occur on a doubly branched state -- `finalizeBlockState`. Successful message execution results in
+writes being committed to `finalizeBlockState`. Note, if message execution fails, state transitions from
 the AnteHandler are persisted.
 
-![DeliverTx](./baseapp_state-deliver_tx.png)
+<!-- ![BeginBlock](./baseapp_state-begin_block.png) -->
```

Comment:
the image should be deleted from the docs and a new image should be made.
I believe @samricotta used draw.io last time, so you should simply be able to open that image in draw.io and edit it, without needed to re-do it from scratch.